README.md

```diff
@@ -42,7 +42,7 @@ cd ${SECOR_INSTALL_DIR}
 ##### Run Secor
 ```sh
 cd ${SECOR_INSTALL_DIR}
-java -ea -Dsecor_group=secor_backup -Dlog4j.configuration=log4j.prod.properties -Dconfig=secor.prod.backup.properties -cp secor-0.1-SNAPSHOT.jar:lib/* com.pinterest.secor.main.ConsumerMain
+java -ea -Dsecor_group=secor_backup -Dlog4j.configuration=log4j.prod.properties -Dconfig=secor.prod.backup.properties -cp secor-0.16-SNAPSHOT.jar:lib/* com.pinterest.secor.main.ConsumerMain
```

Comment:
Could you also directly fix the missings `"` ?

`"secor-0.16-SNAPSHOT.jar:lib/*"`